src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,22 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
+    var date = userData.lastUpdated;
+    var userCountries = Object.keys(userData.institutes);
```

Comment:
Take a look at the diff: line 17 is precisely the same as this line (minus the `.institutes`) part, so instead of simply "adding `.institutes`" to fix the issue, you add an additional step of "moving the method to line 20".

However, this is just a minor issue and I am willing to close one eye for it. I trust that my message of trying your best to minimize the changeset when working on an issue comes across.